### DISCUSSION TITLE
編集中にcssがチラチラ動いて編集しにくい

### DISCUSSION THREAD

**Discussion by duri0214** [increments/qiita-discussions#188] [要望・アイデア・不具合]
（画面は編集中に「a」を打っているだけの動画）
https://drive.google.com/file/d/1Q7G8tXrTxj1R7snYOjRNDCdFF4Z0FrQ0/view?usp=sharing

不具合になるとは思いますが、
ディスプレイの半分ぐらいのブラウザサイズにして、なにかを見ながら編集するようなときに、いつからかはわかりませんが
ちらちら画面が動くようになりました。これが大変目に悪い。。

これはなおせますか？

**Comment by kskwtnk** [comment]
@duri0214 
ご意見ありがとうございます！

確かではありませんが、こちらの問題と同じかもしれません→ https://github.com/increments/qiita-discussions/discussions/181

ご不便をおかけしておりますが、調査中ですので今しばらくお待ちください 🙏 
「これかも」というのは見つかったのですが、若干怪しく…… 🙇

**Comment by ohakutsu** [comment]
@duri0214 
遅くなってしまい、申し訳ございません :bow:

ご報告いただいた状態に近い問題（#181）の修正をしました！
https://github.com/increments/qiita-discussions/discussions/181#discussioncomment-2483383
もし、まだ発生する様でしたら、ご報告していただけると幸いです :pray:
